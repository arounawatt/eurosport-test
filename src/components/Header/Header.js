import { Link } from "gatsby"
import PropTypes from "prop-types"
import React from "react"

import "./Header.scss";

const Header = ({ siteTitle }) => (
  <header className="Header">
      <h1 style={{ margin: 0 }}>
        <Link
          to="/"
          style={{
            color: `white`,
            textDecoration: `none`,
          }}
        >
        <img className="Header__logo" src="https://zephyr-ui.eurosport.com/1.56.0/e6cae33e67433ad1dfcd8af34364b8fc.svg" alt="eurosport" />
        </Link>
      </h1>
  </header>
)

Header.propTypes = {
  siteTitle: PropTypes.string,
}

Header.defaultProps = {
  siteTitle: ``,
}

export default Header
