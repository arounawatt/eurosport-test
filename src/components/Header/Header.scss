.Header {
  display: flex;
  align-items: center;
  background-color: palette(primary);
  padding: 32px;

  .Header__logo {
    width: 200px;

    @include gridle-state(medium) {
      width: 400px;
    }
  }


}