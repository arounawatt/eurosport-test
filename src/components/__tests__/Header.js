import React from "react"
import renderer from "react-test-renderer"
import Header from "../Header/Header"

describe("Header", () => {
  it("should render correctly without props", () => {
    const tree = renderer
      .create(<Header siteTitle="Default Starter" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})