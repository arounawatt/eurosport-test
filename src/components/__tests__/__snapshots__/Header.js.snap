// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly without props 1`] = `
<header
  className="Header"
>
  <h1
    style={
      Object {
        "margin": 0,
      }
    }
  >
    <a
      href="/"
      style={
        Object {
          "color": "white",
          "textDecoration": "none",
        }
      }
    >
      <img
        alt="eurosport"
        className="Header__logo"
        src="https://zephyr-ui.eurosport.com/1.56.0/e6cae33e67433ad1dfcd8af34364b8fc.svg"
      />
    </a>
  </h1>
</header>
`;
